cpp/cgi_progs/zotero_cgi.cc

```diff
@@ -578,7 +578,7 @@ bool ProcessShowQASubActionAdd(const std::multimap<std::string, std::string> &cg
     if (add_type == "global")
         journal_id_to_insert = "NULL";
 
-    db_connection->queryOrDie("INSERT INTO metadata_presence_tracer (zeder_journal_id, marc_field_tag, field_presence) "
+    db_connection->queryOrDie("INSERT INTO metadata_presence_tracer (journal_id, marc_field_tag, field_presence) "
```

Comment:
This won't work anymore with the changes to ub_tools.sql (missing mandatory subfield code in insert statement).

Multiple things need to be changed in zotero_cgi.cc to, and also qa.html (which is missing from the pull request so far).

Reply:
Fixed